test/runtests.jl

```diff
@@ -20,7 +20,29 @@ cd(dirname(@__FILE__)) do
 
     @everywhere include("testdefs.jl")
 
-    reduce(propagate_errors, nothing, pmap(runtests, tests; err_retry=false, err_stop=true))
+    const max_worker_rss = parse(Int, get(ENV, "JULIA_TEST_MAXRSS_MB", "200"))
```

Comment:
It might be too low, but many of the small tests stay below. I just tried a sequence of `backtrace`, `base64`, `bigint`, `ccall`, `cmdlineargs`, `combinatorics` in a single process without exceeding 200MB. That was even in the REPL which probably uses more than script execution.